docs/Objetivo-1/milestones.md

```diff
@@ -0,0 +1,7 @@
+# Milestones
+
+- [M0] Diseño de un módulo que defina la estructura de datos donde vayamos a almacenar libros y géneros. 
+
+- [M1] Diseño de una serie de ficheros que implemente la lógica de negocio y que use el Algoritmo de similitud de Pearson para encontrar libros lo más similar posible al libro proporcionado, según el coeficiente de similitud.
```

Comment:
Los ficheros no se "diseñan". Por favor, escribid con precisión para describir la solución que proponéis a un problema de ingeniería.